api/nodes/permissions.py

```diff
@@ -50,6 +50,20 @@ def has_object_permission(self, request, view, obj):
         return obj.is_public or obj.can_view(auth)
 
 
+class IsAdminAndContributor(permissions.BasePermission):
```

Comment:
Thoughts on the name of this permission? At first I thought `AdminContributor` but those are different things already aren't they!